uni/lib/view/bug_report/widgets/form.dart

```diff
@@ -322,10 +321,4 @@ class BugReportFormState extends State<BugReportForm> {
         .loadString(assetsPath)
```

Comment:
I share @bdmendes opinion. I think it is safe to remove this method, it isn't used anywhere else.
https://github.com/NIAEFEUP/uni/blob/992bd89b62ecb3ff96bcc89a5d2ed69991c5de47/uni/lib/view/bug_report/widgets/form.dart#L319-L323

Reply:
done!